xain_fl/__main__.py

```diff
@@ -23,7 +25,6 @@
 def main():
     """Start a coordinator instance
     """
-    initialize_logging()
```

Comment:
Won't we lose logs related to configuration if we initialize logging _after_ loading the config

Reply:
Good point, I was concerned about this too. 
The way we configure/use structlog is that we use structlog as a wrapper around `logger`. That means in this case that we will still get following the log message:

 `2020-02-11 11:07.44 Invalid config                 error=configs/no_config.toml not found` 

(since we are using `logging` directly instead of `from structlog import get_logger`) but we won't be able to get this message formatted in JSON anymore.

Reply:
ah this is nice. I think this is perfectly acceptable indeed.